### ISSUE TITLE
[BUG] assert referenced_declaration is not None

### ISSUE THREAD

**Issue by PFAhard** [Ackee-Blockchain/tools-for-solidity-vscode#65]
```python
  File "C:\Users\pfapostol\AppData\Local\Programs\Python\Python38\lib\site-packages\woke\lsp\server.py", line 268, in _task_done_callback
    task.result()
  File "C:\Users\pfapostol\AppData\Local\Programs\Python\Python38\lib\site-packages\woke\lsp\lsp_compiler.py", line 187, in run
    await self.__compilation_loop()
  File "C:\Users\pfapostol\AppData\Local\Programs\Python\Python38\lib\site-packages\woke\lsp\lsp_compiler.py", line 977, in __compilation_loop
    await self.__compile(self.__discovered_files)
  File "C:\Users\pfapostol\AppData\Local\Programs\Python\Python38\lib\site-packages\woke\lsp\lsp_compiler.py", line 876, in __compile
    await self.__compile(files_to_recompile, False)
  File "C:\Users\pfapostol\AppData\Local\Programs\Python\Python38\lib\site-packages\woke\lsp\lsp_compiler.py", line 848, in __compile
    self.__ir_reference_resolver.run_post_process_callbacks(
  File "C:\Users\pfapostol\AppData\Local\Programs\Python\Python38\lib\site-packages\woke\ast\ir\reference_resolver.py", line 174, in run_post_process_callbacks
    callback(callback_params)
  File "C:\Users\pfapostol\AppData\Local\Programs\Python\Python38\lib\site-packages\woke\ast\ir\meta\import_directive.py", line 165, in _post_process
    assert referenced_declaration is not None
AssertionError
```

**Comment by PFAhard** [comment]
https://github.com/code-423n4/2023-10-wildcat

When you try the ctrl+cliсk function, but sometimes it’s just like that.

**Comment by michprev** [comment]
This is the same issue as https://github.com/Ackee-Blockchain/woke/issues/274. The problem is overloaded functions imported in import directives.

**Comment by michprev** [comment]
The issue should be fixed in Tools for Solidity 1.10.4.
